### DISCUSSION TITLE
0.0.2308.121

### DISCUSSION THREAD

**Discussion by prosthetichead** [prosthetichead/GarlicPress#72] [Announcements]
## What's Changed
* Fixed selected drive being backed up twice 
* Images now work when system folders are on root of sd card 2

**Full Changelog**: https://github.com/prosthetichead/GarlicPress/compare/0.0.2308.071...0.0.2308.121

<hr /><em>This discussion was created from the release <a href='https://github.com/prosthetichead/GarlicPress/releases/tag/0.0.2308.121'>0.0.2308.121</a>.</em>

**Comment by TotallyBoard** [comment]
Hello! I'm brand new and getting an error when trying to run the GarlicPress.exe. It says "Error: The adb.exe executable could not be found at adb.exe". Any idea how to fix this?

**Reply by shawnanderson** [reply]
Are you running the code from within Visual Studio?  If so, I had to copy adb.exe, AdbWinApi.dll, and AdbWinUsbApi.dll in the bin\Debug\net6.0-windows7.0 folder.

**Reply by prosthetichead** [reply]
@TotallyBoard are you running the release version or the source code?
The release version should include all the files in the zip.

**Reply by TotallyBoard** [reply]
Oh I have no idea. I just downloaded the one called GarlicPress.0.0.2308.121 and tried opening it and it just crashes :(

**Reply by prosthetichead** [reply]
That's really odd adb.exe is included in the zip did you extract everything and run it can you see adb.exe with the other files after extracting the zip file?

**Comment by dmnthe** [comment]
Can you make Garlic Press support "unknown" ROMs folders?

**Reply by prosthetichead** [reply]
what would be an "Unknown" rom folder? Do you mean like just any folder you make with any name, does garlicOS even support that?
at the moment it uses the system folders to know which platform it is and what art to get off screen scraper.

**Reply by dmnthe** [reply]
I mean "Banpresto" "Cave" "Dataeast" "irem" "konami" "midway" "namco" "psikyo" "sega" "taito" "tate" "toaplan" "visco"

**Reply by dmnthe** [reply]
https://archive.org/details/tiny-best-set-go-extra-unofficial

**Reply by prosthetichead** [reply]
I've added this to issue #84

**Reply by dmnthe** [reply]
Thank you 🙏

**Comment by TotallyBoard** [comment]
> That's really odd adb.exe is included in the zip did you extract everything and run it can you see adb.exe with the other files after extracting the zip file?

I just download the file and it opens with 7zip and just freaks out. Might just be something on my end? Idk

**Reply by prosthetichead** [reply]
where did you extract the zip file too? is ADB.exe still in the folder when you extract everything?

**Comment by prosthetichead** [comment]
This Version had a major bug in it. If you had issues please update

**Reply by TotallyBoard** [reply]
Hey! So now, GarlicPress is working and it is recognizing my RG35XX. However, when I try to update game art, I get this weird error. It says: "Unhandled exception has occurred in your aplication. If you click Continue, the application will ignore this error and attempt to continue. If you click quit, the application will close immediately. 

The directory assets\temp of the filename assets/temp/gameart-up.png does not exist"

Any idea what that is? I'm also trying to follow your YouTube video, but I may just be too much of a noob LOL

**Reply by prosthetichead** [reply]
Sorry looks like I missed something else! I will take a look tonight when I get home from work. No I think the issue is 100% my side. I'm working on a largish update and so somehow I've messed up this hotfix I did on Friday night. Note to self don't do release late at night on a Friday lol.

I should have it fixed for you tonight.

**Reply by TotallyBoard** [reply]
OK cool! Excited to try it out

**Reply by prosthetichead** [reply]
@TotallyBoard   All fixed in the latest release
